template/template_example.Rmd

```diff
@@ -62,42 +66,46 @@ if (!dir.exists(results_dir)) {
 In the same place you put this `.Rmd` file, you should now have three new empty folders called `data`, `plots`, and `results`!
 
 ## Obtain the dataset from refine.bio
+<!-- ADD LINK, THEN LEAVE THIS SECTION AS IS -->
 
 For general information about downloading data for these examples, see our ['Getting Started' section](https://alexslemonade.github.io/refinebio-examples/01-getting-started/getting-started.html#how-to-get-the-data). 
 
 Go to this [dataset's page on refine.bio](<<REFINEBIO DATASET PAGE LINK>>).
 
 Click the "Download Now" button on the right side of this screen.  
 
-<img src="screenshots/download-now.png" width=200>  
+<img src="https://raw.githubusercontent.com/AlexsLemonade/refinebio-examples/master/template/screenshots/download-now.png" width=200>  
 
 Fill out the pop up window with your email and our Terms and Conditions:  
 
-<img src="screenshots/download-email.png" width=500>  
+<img src="https://raw.githubusercontent.com/AlexsLemonade/refinebio-examples/master/template/screenshots/download-email.png" width=500>  
 
 It make take a few minutes for the dataset to process.
 You will get an email when it is ready. 
 
 ## About the dataset we are using for this example
+<!-- ADD PATH AND SUMMARY -->
 
-For this example analysis, we will use this [<<BRIEF DATASET NAME>>](https://alexslemonade.github.io/refinebio-examples/ <<RELATIVE PATH TO THIS ANALYSIS FROM REFINEBIO-EXAMPLES>> )..
+For this example analysis, we will use this [<!--BRIEF DATASET NAME-->](https://alexslemonade.github.io/refinebio-examples/ <!--RELATIVE PATH TO THIS ANALYSIS FROM REFINEBIO-EXAMPLES--> )..
```

Comment:
Did you test this? I am not sure if you can put a comment inside a markdown url... 

```suggestion
For this example analysis, we will use this [<<BRIEF DATASET NAME>>](https://alexslemonade.github.io/refinebio-examples/ <!--RELATIVE PATH TO THIS ANALYSIS FROM REFINEBIO-EXAMPLES--> )..
```